tools/accuracy_checker/accuracy_checker/adapters/pose_estimation.py

```diff
@@ -101,25 +105,33 @@ def process(self, raw, identifiers, frame_meta):
             pad = meta.get('padding', [0, 0, 0, 0])
             transpose_order = (1, 2, 0) if heatmap.shape[0] == 19 else (0, 1, 2)
 
+            s = 8
+
             heatmap = np.transpose(np.squeeze(heatmap), transpose_order)
-            heatmap = cv2.resize(heatmap, (0, 0), fx=8, fy=8, interpolation=cv2.INTER_CUBIC)
+            heatmap = cv2.resize(heatmap, (0, 0), fx=s, fy=s, interpolation=cv2.INTER_CUBIC)
             heatmap = heatmap[pad[0]:heatmap.shape[0] - pad[2], pad[1]:heatmap.shape[1] - pad[3]:, :]
-            heatmap = cv2.resize(heatmap, (width, height), interpolation=cv2.INTER_CUBIC)
-            heatmap_avg = heatmap_avg + heatmap
+            # heatmap = cv2.resize(heatmap, (width, height), interpolation=cv2.INTER_CUBIC)
+            # heatmap_avg = heatmap_avg + heatmap
+            heatmap_avg = heatmap
 
             paf = np.transpose(np.squeeze(paf), transpose_order)
-            paf = cv2.resize(paf, (0, 0), fx=8, fy=8, interpolation=cv2.INTER_CUBIC)
+            paf = cv2.resize(paf, (0, 0), fx=s, fy=s, interpolation=cv2.INTER_CUBIC)
             paf = paf[pad[0]:paf.shape[0] - pad[2], pad[1]:paf.shape[1] - pad[3], :]
-            paf = cv2.resize(paf, (width, height), interpolation=cv2.INTER_CUBIC)
-            paf_avg = paf_avg + paf
+            # paf = cv2.resize(paf, (width, height), interpolation=cv2.INTER_CUBIC)
```

Comment:
please return adapter in the previous state or remove debug comments, but in the second case, can you please guarantee that adapter behavior is not changed?

Reply:
Yeah, this change was unintended. Reverted back to the original implementation.